perfkitbenchmarker/benchmarks/cassandra_stress_benchmark.py

```diff
@@ -140,201 +109,86 @@ def Prepare(benchmark_spec):
+  """Run Cassandra-stress test on loader node.
 
   Args:
-    benchmark_spec: The benchmark specification. Contains all data
-        that is required to run the benchmark.
     vm: The target vm.
-
-  Returns:
-    A boolean indicates the cluster is ready.
-  """
-  logging.info('Verify Cluster')
-  num_expected_data_nodes = len(benchmark_spec.vm_dict[DATA_NODE])
-  ret, _ = vm.RemoteCommand(
-      '%s/bin/nodetool status | grep UN | wc -l' % CASSANDRA_DIR)
-  num_actual_data_nodes = int(ret)
-  if num_actual_data_nodes != num_expected_data_nodes:
-    logging.warning('Expecting %s data nodes, but only %s are up.',
-                    num_expected_data_nodes,
-                    num_actual_data_nodes)
-  return num_actual_data_nodes == num_expected_data_nodes
-
-
-def VerifyNode(vm, seed_vm):
-  """Verify Cassandra is setup on the target vm.
-
-  Verify Cassandra is setup on the target vm.  If not, try to start Cassandra
-  on the VM again.
-
-  Args:
-    vm: The target vm.
-    seed_vm: The first vm in data nodes.
-  """
-  ret, _ = vm.RemoteCommand(
-      'ps -e | grep java | awk "{print $1}"')
-  if not ret:
-    logging.warning('Restarting Cassandra on %s', vm.hostname)
-    StartCassandraDataNodeServer(vm, seed_vm)
-
-
-def RunTestOnLoader(vm, data_nodes_ip_addresses):
-  """Running Cassandra-stress test on loader node.
-
-  Cassandra-stress tool page:
-  http://www.datastax.com/documentation/cassandra/2.0/cassandra/tools/toolsCStress_t.html
-
-  Args:
-    vm: The target vm.
-    data_nodes_ip_addresses: Ip addresses for all data nodes seperated by ','.
+    data_node_ips: List of IP addresses for all data nodes.
   """
   vm.RemoteCommand(
-      './%s/tools/bin/cassandra-stress '
-      '--file "./%s.results" --nodes %s '
+      '%s '
+      '--file "%s" --nodes %s '
       '--replication-factor %s --consistency-level %s '
       '--num-keys %s -K %s -t %s' % (
-          CASSANDRA_DIR, vm.hostname, data_nodes_ip_addresses,
+          CASSANDRA_STRESS,
+          _ResultFilePath(vm),
+          ','.join(data_node_ips),
           REPLICATION_FACTOR, CONSISTENCY_LEVEL, FLAGS.num_keys,
           RETRIES, FLAGS.num_cassandra_stress_threads))
 
 
-def InsertTest(benchmark_spec, vm):
+def RunCassandraStress(benchmark_spec):
   """Start Cassandra test.
 
   Args:
     benchmark_spec: The benchmark specification. Contains all data
         that is required to run the benchmark.
-    vm: The target vm.
   """
   logging.info('Creating Keyspace.')
-  data_nodes_ip_addresses = [
-      data_vm.internal_ip
-      for data_vm in benchmark_spec.vm_dict[DATA_NODE]]
-  data_nodes_ip_addresses = ','.join(data_nodes_ip_addresses)
-  vm.RemoteCommand(
-      './%s/tools/bin/cassandra-stress '
+  data_node_ips = [data_vm.internal_ip
+                   for data_vm in benchmark_spec.vm_dict[DATA_NODE]]
+  loader_vms = benchmark_spec.vm_dict[LOADER_NODE]
+  loader_vms[0].RemoteCommand(
+      '%s '
       '--nodes %s --replication-factor %s '
```

Comment:
Has the stress command changed? I remember trying to upgrade the package a while ago, but the commands seem to be a bit different.